Cargo.toml

```diff
@@ -1,26 +1,28 @@
 [package]
 authors = ["w3irdrobot <rob@w3ird.tech>"]
 categories = ["embedded", "hardware-support", "no-std"]
-description = "Platform-agnostic Rust driver for the Maxim DS4432 Dual-Channel, I^2C, 7-Bit Sink/Source Current Digital To Analog (DAC) converter."
+description = "Platform-agnostic Rust driver for the Maxim DS4432 Dual-Channel, I2C, 7-Bit Sink/Source Current Digital To Analog (DAC) converter."
 documentation = "https://docs.rs/ds4432"
 edition = "2021"
 keywords = ["no-std", "embedded-hal-driver", "dac"]
 license = " AGPL-3.0-only"
 name = "ds4432"
 repository = "https://github.com/w3irdrobot/ds4432"
-version = "0.1.0"
+rust-version = "1.71.1"
+version = "0.2.0"
 
 [dependencies]
-defmt = { version = "0.3.8", optional = true }
-embedded-hal = { version = "1.0.0", optional = true }
-embedded-hal-async = { version = "1.0.0", optional = true }
-maybe-async-cfg = "0.2.4"
+defmt = { version = "0.3", optional = true }
+embedded-hal = { version = "1.0" }
```

Comment:
A good practice is to restrict some code with a feature only to save build time (for bigger lib) because at the end of the day the dead code will not end in the binary anyway.

Reply:
ok, I see, reverted